models/classes/Platform/Repository/Lti1p3RegistrationRepository.php

```diff
@@ -41,25 +41,26 @@
 class Lti1p3RegistrationRepository extends ConfigurableService implements RegistrationRepositoryInterface
 {
     private const OIDC_URL = ROOT_URL . 'taoLti/Security/oidc';
+    private const OAUTH_URL = 'http://tao-nginx/taoLti/Security/oauth';
```

Comment:
This must be removed.

Reply:
both of them? We can make Platform factory or I can move it to config?

Reply:
This is built on the fly. @boajer  is refactoring this to have a platform repository, cause this information will be need in other place. I think you should talk to him to converge in one solution, but still `http://tao-nginx/taoLti/Security/oauth` is not a correct value.

Reply:
Is this fixing that comment? https://github.com/oat-sa/extension-tao-lti/commit/a9ef98075a22ae675892b3bddd52d9d4800ae260